firestore-send-email/functions/src/helpers.ts

```diff
@@ -0,0 +1,31 @@
+import { createTransport } from "nodemailer";
+
+export const setSmtpCredentials = (config) => {
+  const {
+    smtpConnectionUri,
+    smtpServerDomain, // eg.smtp.gmail.com:465
+    smtpServerSSL,
+    smtpEmail,
+    smtpPassword,
+  } = config;
+  let smtpCredentials;
```

Comment:
We can perhaps return early from this function. Instead of assigning the credentials to a variable.

`if(smtpServerDomaiin) return createTransport(...`
`if(smtpConnectionUri) return createTransport(...`
`return null;`